src/main/scala/util/gen.scala

```diff
@@ -119,8 +120,8 @@ object gen {
                                           ): Expr => String =
         functionFromExpr(name, gen) andThen
           GAP8.Module.injectUnpacking andThen
-          C.Module.translateToString andThen
-          run(SyntaxChecker(_))
+          C.Module.translateToString// andThen
+          //run(SyntaxChecker(_))
```

Comment:
Can this be commented back in? Or alternative, delete it completely.

Reply:
Sure, I can't remember why I kicked that out in the first place. Anyways, I introduced it again with a hint that it should be done more thoroughly (this syntax checks only accelerator function + some declarations)